lib/HALTarget/IMU.cpp

```diff
@@ -94,7 +108,7 @@ void IMU::readAccelerometer()
 {
     m_imuDrv.readAcc();
     m_accelerometerValues.valueX = m_imuDrv.a.x - m_rawAccelerometerOffsetX;
-    m_accelerometerValues.valueY = m_imuDrv.a.y - m_rawAccelerometerOffsetY;
+    m_accelerometerValues.valueY = m_imuDrv.a.y;

```

Comment:
It is still read even when disabled?

Reply:
m_imuDrv.a.y just delivers 0 if the axis is disabled. If somebody else wants to use another axis of the IMU, less modificatios are necessary. Would you remove reading all the non-used axes?

Reply:
You already removed all other structures that support the use of the y axis. I would set this value to 0 in order to prevent any wrong data comming in.  I would also document it in the doxygen description of this method that the value is invalid given that the axis is disabled and it will always return 0.